sycl/include/CL/sycl/INTEL/online_compiler.hpp

```diff
@@ -61,7 +61,7 @@ class device_arch {
 class online_compile_error : public sycl::exception {
 public:
   online_compile_error() = default;
-  online_compile_error(const string_class &Msg) : sycl::exception(Msg) {}
+  online_compile_error(const std::string &Msg) : sycl::exception(Msg) {}
```

Comment:
is this a NOP change? does it not affect ABI?

Reply:
Aliases do no participate in name mangling, so this change won't affect ABI